nix/packages.nix

```diff
@@ -7,7 +7,28 @@
     version = "0.4.0";
     packageJSON = cardano-graphql-src + "/package.json";
     yarnLock = cardano-graphql-src + "/yarn.lock";
-    src = cardano-graphql-src;
+    src = lib.cleanSourceWith {
+      filter = name: type: let
+        baseName = baseNameOf (toString name);
+        sansPrefix = lib.removePrefix (toString ../.) name;
+        in_blacklist =
+          lib.hasPrefix "/node_modules" sansPrefix ||
+          lib.hasPrefix "/build" sansPrefix;
+        in_whitelist =
+          (type == "directory") ||
+          (lib.hasSuffix ".yml" name) ||
+          (lib.hasSuffix ".ts" name) ||
+          (lib.hasSuffix ".json" name) ||
+          (lib.hasSuffix ".graphql" name) ||
+          (lib.hasPrefix "/source" sansPrefix) ||
```

Comment:
```suggestion
          (lib.hasPrefix "/src" sansPrefix) ||
```

Reply:
I'll just remove it, obviously not needed since it builds without all of src being whitelisted.